### ISSUE TITLE
Hippo 12.5 does not start on PCF

### ISSUE THREAD

**Issue by fdeberle** [woonsanko/hippo-on-spring-boot#4]
Hi @woonsanko,

have you tried using the current version 12.5 of Hippo?

A `cf push` on Pivotal Cloud Foundry is resulting in errors on startup:

```
jvmkill killing current process
Exit status 137
```

Complete Log:
```
2018-11-20T11:46:25.764+01:00 [APP/PROC/WEB/0] [OUT] 2018-11-20 10:46:25.764 INFO 17 --- [ main] s.b.c.e.t.TomcatEmbeddedServletContainer : Tomcat started on port(s): 8080 (http)
2018-11-20T11:46:25.967+01:00 [CELL/0] [OUT] Container became healthy
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-Acceptor-0" #80 daemon prio=5 os_prio=0 tid=0x00007fe42c4d1800 nid=0x10a runnable [0x00007fe40139c000]
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: RUNNABLE
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.ServerSocketChannelImpl.accept0(Native Method)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:422)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d085d8> (a java.lang.Object)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.net.NioEndpoint$Acceptor.run(NioEndpoint.java:443)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-ClientPoller-1" #79 daemon prio=5 os_prio=0 tid=0x00007fe3fe5f4800 nid=0x109 runnable [0x00007fe40149d000]
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:93)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d08b88> (a sun.nio.ch.Util$3)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d08b78> (a java.util.Collections$UnmodifiableSet)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d08930> (a sun.nio.ch.EPollSelectorImpl)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.net.NioEndpoint$Poller.run(NioEndpoint.java:775)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-ClientPoller-0" #78 daemon prio=5 os_prio=0 tid=0x00007fe42d230000 nid=0x108 runnable [0x00007fe40159e000]
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d09470> (a sun.nio.ch.Util$3)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d09460> (a java.util.Collections$UnmodifiableSet)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d09328> (a sun.nio.ch.EPollSelectorImpl)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] "NioBlockingSelector.BlockPoller-1" #77 daemon prio=5 os_prio=0 tid=0x00007fe42ef3b800 nid=0x107 runnable [0x00007fe40169f000]
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d09ca0> (a sun.nio.ch.Util$3)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d09c90> (a java.util.Collections$UnmodifiableSet)
2018-11-20T11:47:15.544+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x0000000723d09b78> (a sun.nio.ch.EPollSelectorImpl)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.net.NioBlockingSelector$BlockPoller.run(NioBlockingSelector.java:339)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] "container-0" #76 prio=5 os_prio=0 tid=0x00007fe4045c0000 nid=0x106 waiting on condition [0x00007fe401da0000]
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (sleeping)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.sleep(Native Method)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.core.StandardServer.await(StandardServer.java:427)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedServletContainer$1.run(TomcatEmbeddedServletContainer.java:174)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] "ContainerBackgroundProcessor[StandardEngine[Tomcat]]" #75 daemon prio=5 os_prio=0 tid=0x00007fe42f0b1000 nid=0x105 waiting on condition [0x00007fe401ea1000]
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.run(ContainerBase.java:1355)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] "pool-3-thread-1" #74 prio=5 os_prio=0 tid=0x00007fe404a3f800 nid=0xf9 waiting on condition [0x00007fe4023a2000]
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: WAITING (parking)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at sun.misc.Unsafe.park(Native Method)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x00000007237c6090> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
2018-11-20T11:47:15.545+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x00000007236f2bc8> (a java.util.concurrent.ConcurrentHashMap)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveManagedList(BeanDefinitionValueResolver.java:382)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:157)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1486)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1231)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:551)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveInnerBean(BeanDefinitionValueResolver.java:299)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:122)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveManagedMap(BeanDefinitionValueResolver.java:407)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:165)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.ConstructorResolver.resolveConstructorArguments(ConstructorResolver.java:648)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:145)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1148)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1050)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:512)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:754)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:866)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:542)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000072371ded0> (a java.lang.Object)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.hippoecm.hst.site.container.SpringComponentManager.initialize(SpringComponentManager.java:151)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.hippoecm.hst.site.container.DefaultHstSiteConfigurer.initializeComponentManager(DefaultHstSiteConfigurer.java:275)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x00000007208e4fb8> (a org.hippoecm.hst.site.container.DefaultHstSiteConfigurer)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.hippoecm.hst.site.container.DefaultHstSiteConfigurer.lambda$initialize$0(DefaultHstSiteConfigurer.java:235)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at org.hippoecm.hst.site.container.DefaultHstSiteConfigurer$$Lambda$10/1043550205.run(Unknown Source)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] "commons-pool-EvictionTimer" #22 daemon prio=5 os_prio=0 tid=0x00007fe42e75f800 nid=0x56 in Object.wait() [0x00007fe40ba35000]
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (on object monitor)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Native Method)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.mainLoop(Timer.java:552)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x00000007205e65f8> (a java.util.TaskQueue)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.run(Timer.java:505)
2018-11-20T11:47:15.548+01:00 [APP/PROC/WEB/0] [OUT] "MVStore background writer nio:/home/vcap/app/repository/repodb/repodb.mv.db" #20 daemon prio=5 os_prio=0 tid=0x00007fe42fc88800 nid=0x55 in Object.wait() [0x00007fe40b934000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Native Method)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at org.h2.mvstore.MVStore$BackgroundWriterThread.run(MVStore.java:2708)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000072047b980> (a java.lang.Object)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "Abandoned connection cleanup thread" #15 daemon prio=5 os_prio=0 tid=0x00007fe42f8c0800 nid=0x51 in Object.wait() [0x00007fe40d6e0000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (on object monitor)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x00000007205e6610> (a java.lang.ref.ReferenceQueue$Lock)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at com.mysql.cj.jdbc.AbandonedConnectionCleanupThread.run(AbandonedConnectionCleanupThread.java:70)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "localhost-startStop-1" #14 daemon prio=5 os_prio=0 tid=0x0000000002bf5000 nid=0x50 waiting on condition [0x00007fe40c82c000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (parking)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at sun.misc.Unsafe.park(Native Method)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x0000000720681240> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2078)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.LinkedBlockingQueue.poll(LinkedBlockingQueue.java:467)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1073)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "Tomcat-startStop-1" #13 daemon prio=5 os_prio=0 tid=0x00007fe42e1d1000 nid=0x4f waiting on condition [0x00007fe40cd32000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x00000007203fa1c0> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "Service Thread" #9 daemon prio=9 os_prio=0 tid=0x00007fe42c0ec800 nid=0x40 runnable [0x0000000000000000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: RUNNABLE
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "C1 CompilerThread3" #8 daemon prio=9 os_prio=0 tid=0x00007fe42c0d8000 nid=0x3f waiting on condition [0x0000000000000000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "C2 CompilerThread2" #7 daemon prio=9 os_prio=0 tid=0x00007fe42c0cd800 nid=0x3e waiting on condition [0x0000000000000000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "C2 CompilerThread1" #6 daemon prio=9 os_prio=0 tid=0x00007fe42c0cb800 nid=0x3d waiting on condition [0x0000000000000000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "C2 CompilerThread0" #5 daemon prio=9 os_prio=0 tid=0x00007fe42c0c8800 nid=0x3c waiting on condition [0x0000000000000000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "Signal Dispatcher" #4 daemon prio=9 os_prio=0 tid=0x00007fe42c0c7000 nid=0x3b waiting on condition [0x0000000000000000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "Finalizer" #3 daemon prio=8 os_prio=0 tid=0x00007fe42c094800 nid=0x3a in Object.wait() [0x00007fe410a3b000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: WAITING (on object monitor)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000071fa1a4d8> (a java.lang.ref.ReferenceQueue$Lock)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:165)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:216)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "Reference Handler" #2 daemon prio=10 os_prio=0 tid=0x00007fe42c08a800 nid=0x39 in Object.wait() [0x00007fe410b3c000]
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Object.java:502)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000071fa16550> (a java.lang.ref.Reference$Lock)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "VM Thread" os_prio=0 tid=0x00007fe42c082800 nid=0x38 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#0 (ParallelGC)" os_prio=0 tid=0x00007fe42c022000 nid=0x2e runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#1 (ParallelGC)" os_prio=0 tid=0x00007fe42c024000 nid=0x30 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#2 (ParallelGC)" os_prio=0 tid=0x00007fe42c025800 nid=0x31 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#3 (ParallelGC)" os_prio=0 tid=0x00007fe42c027800 nid=0x32 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#4 (ParallelGC)" os_prio=0 tid=0x00007fe42c029000 nid=0x33 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#5 (ParallelGC)" os_prio=0 tid=0x00007fe42c02b000 nid=0x34 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#6 (ParallelGC)" os_prio=0 tid=0x00007fe42c02c800 nid=0x35 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "GC task thread#7 (ParallelGC)" os_prio=0 tid=0x00007fe42c02e800 nid=0x36 runnable
2018-11-20T11:47:15.549+01:00 [APP/PROC/WEB/0] [OUT] "VM Periodic Task Thread" os_prio=0 tid=0x00007fe42c0ef000 nid=0x41 waiting on condition
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] JNI global references: 1337099
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] Heap
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] PSYoungGen total 1104384K, used 21491K [0x00000007b1a00000, 0x00000007fa680000, 0x00000007faa00000)
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] eden space 1017344K, 2% used [0x00000007b1a00000,0x00000007b2efcc88,0x00000007efb80000)
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] from space 87040K, 0% used [0x00000007f5180000,0x00000007f5180000,0x00000007fa680000)
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] to space 87552K, 0% used [0x00000007efb80000,0x00000007efb80000,0x00000007f5100000)
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] ParOldGen total 563712K, used 84717K [0x000000071fa00000, 0x0000000742080000, 0x00000007b1a00000)
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] object space 563712K, 15% used [0x000000071fa00000,0x0000000724cbb528,0x0000000742080000)
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] Metaspace used 92449K, capacity 93372K, committed 94268K, reserved 1132544K
2018-11-20T11:47:15.590+01:00 [APP/PROC/WEB/0] [OUT] class space used 10595K, capacity 10809K, committed 10880K, reserved 1048576K
2018-11-20T11:47:20.544+01:00 [APP/PROC/WEB/0] [ERR] jvmkill killing current process
2018-11-20T11:47:20.638+01:00 [APP/PROC/WEB/0] [OUT] Exit status 137
2018-11-20T11:47:20.684+01:00 [API/3] [OUT] Process has crashed with type: "web"
```

manifest.yml:
```
---
applications:
- name: wbcms-be
  memory: 4G
  disk_quota: 2G
  timeout: 180
  instances: 1
  buildpack: java_buildpack_offline
  path: target/wbcms-be-0.1.0-SNAPSHOT.jar
  env:
    JBP_CONFIG_TOMCAT: '{ tomcat: { version: 8.5.+ } }'
    JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '[enabled: false]'
    SPRING_PROFILES_ACTIVE: 'dev'
```

Any help would be greatly appreciated!

**Comment by woonsanko** [comment]
Hi @fdeberle ,

Did you apply the "Deferred Initialization of RepositoryServlet" option to your project?
Ref: https://github.com/woonsanko/hippo-on-spring-boot#option-for-deferred-initialization-of-repositoryservlet

If the repository servlet took too much time to initialize (e.g, building lucene index from the scratch), PCF could have decided to kill the tomcat after the timeout.

Regards,

Woonsan

**Comment by fdeberle** [comment]
Hi @woonsanko,

good point. Could you please check the link? http://hipshoot.forge.onehippo.org/apidocs/org/onehippo/forge/hipshoot/spring/boot/support/servlet/DeferredInitDelegatingServlet.html is being redirected to https://bloomreach-forge.github.io/

**Comment by fdeberle** [comment]
This one? https://bloomreach-forge.github.io/hipshoot/apidocs/org/onehippo/forge/hipshoot/spring/boot/support/servlet/DeferredInitDelegatingServlet.html

**Comment by woonsanko** [comment]
Yes, @fdeberle . I've just fixed the links in the README.md:
- https://bloomreach-forge.github.io/hipshoot/hipshoot-spring-boot-support/index.html
- https://bloomreach-forge.github.io/hipshoot/apidocs/org/onehippo/forge/hipshoot/spring/boot/support/servlet/DeferredInitDelegatingServlet.html

Thanks!

**Comment by fdeberle** [comment]
So I changed
```
  <!-- Repository -->
  <servlet>
    <servlet-name>Repository</servlet-name>
    <servlet-class>org.hippoecm.repository.RepositoryServlet</servlet-class>
    <load-on-startup>4</load-on-startup>
  </servlet>
```
to
```
  <!-- Repository -->
  <servlet>
    <servlet-name>Repository</servlet-name>
    <servlet-class>org.onehippo.forge.hipshoot.spring.boot.support.servlet.DeferredInitDelegatingServlet</servlet-class>
    <init-param>
      <param-name>DeferredInitDelegatingServlet.delegateServletClass</param-name>
      <param-value>org.hippoecm.repository.RepositoryServlet</param-value>
    </init-param>
    <load-on-startup>100</load-on-startup>
  </servlet>
```
but I still get errors :(

```
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:474)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.valves.RemoteIpValve.invoke(RemoteIpValve.java:677)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:798)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1434)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076e81efe8> (a org.apache.tomcat.util.net.NioEndpoint$NioSocketWrapper)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-exec-4" #47 daemon prio=5 os_prio=0 tid=0x00007f0534011000 nid=0xc3 waiting on condition [0x00007f04ff1fa000]
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: WAITING (parking)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at sun.misc.Unsafe.park(Native Method)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x000000076c4176c8> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.threads.TaskQueue.take(TaskQueue.java:103)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.threads.TaskQueue.take(TaskQueue.java:31)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-exec-3" #46 daemon prio=5 os_prio=0 tid=0x00007f053400e800 nid=0xb7 waiting on condition [0x00007f04ff2fb000]
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-exec-2" #45 daemon prio=5 os_prio=0 tid=0x00007f053400d800 nid=0xb6 waiting on condition [0x00007f05785e5000]
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] "EventListenersContainerSessionChecker-1" #44 daemon prio=5 os_prio=0 tid=0x00007f054d546000 nid=0xab in Object.wait() [0x00007f04ff3fc000]
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (on object monitor)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Native Method)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.hippoecm.hst.core.jcr.EventListenersContainerImpl$EventListenersContainerSessionChecker.run(EventListenersContainerImpl.java:433)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076e81f698> (a org.hippoecm.hst.core.jcr.EventListenersContainerImpl$EventListenersContainerSessionChecker)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] "Statistics Thread-/site_ehCacheManager-1" #43 daemon prio=5 os_prio=0 tid=0x00007f054c3bb000 nid=0xaa waiting on condition [0x00007f04ffcfd000]
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (parking)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x000000076e285d20> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-exec-5" #48 daemon prio=5 os_prio=0 tid=0x00007f0534012800 nid=0xce runnable [0x00007f04ff0f5000]
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: RUNNABLE
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ClassLoader.defineClass1(Native Method)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
2018-11-23T15:41:15.417+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.catalina.loader.WebappClassLoaderBase.findClassInternal(WebappClassLoaderBase.java:2277)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2078)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1093)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:809)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1074)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] "/site_ehCacheManager" #41 daemon prio=5 os_prio=0 tid=0x00007f054c39e800 nid=0xa8 in Object.wait() [0x00007f04ffdfe000]
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: WAITING (on object monitor)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Native Method)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Object.java:502)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.mainLoop(Timer.java:526)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076e2fb008> (a java.util.TaskQueue)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.run(Timer.java:505)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] "Timer-0" #40 daemon prio=5 os_prio=0 tid=0x00007f054c2e9800 nid=0xa7 in Object.wait() [0x00007f055330b000]
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (on object monitor)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.mainLoop(Timer.java:552)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076e285f38> (a java.util.TaskQueue)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] "pool-3-thread-1" #39 prio=5 os_prio=0 tid=0x00007f0540bc4000 nid=0xa6 waiting on condition [0x00007f0550919000]
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: WAITING (parking)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at sun.misc.Unsafe.park(Native Method)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x000000076dab62b8> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] "http-nio-8080-exec-1" #30 daemon prio=5 os_prio=0 tid=0x00007f0534018800 nid=0x9d waiting on condition [0x00007f0552220000]
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] - parking to wait for <0x000000076c4176c8> (a java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2039)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.threads.TaskQueue.take(TaskQueue.java:103)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.threads.TaskQueue.take(TaskQueue.java:31)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] "ObservationManager" #29 daemon prio=5 os_prio=0 tid=0x00007f05404d1000 nid=0x92 in Object.wait() [0x00007f0552521000]
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.commons.collections.buffer.BlockingBuffer.remove(BlockingBuffer.java:192)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076c3d2378> (a org.apache.commons.collections.buffer.BlockingBuffer)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] at org.apache.jackrabbit.core.observation.ObservationDispatcher.run(ObservationDispatcher.java:153)
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] "Jackrabbit-ClusterRevisionJanitor" #27 daemon prio=5 os_prio=0 tid=0x00007f054046b000 nid=0x90 waiting on condition [0x00007f0552622000]
2018-11-23T15:41:15.418+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (sleeping)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.mainLoop(Timer.java:552)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076b6acaa8> (a java.util.TaskQueue)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.util.TimerThread.run(Timer.java:505)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "Abandoned connection cleanup thread" #13 daemon prio=5 os_prio=0 tid=0x00007f0560117800 nid=0x46 in Object.wait() [0x00007f059c8f7000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: TIMED_WAITING (on object monitor)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Native Method)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:144)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076b5c0778> (a java.lang.ref.ReferenceQueue$Lock)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at com.mysql.cj.jdbc.AbandonedConnectionCleanupThread.run(AbandonedConnectionCleanupThread.java:70)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Thread.run(Thread.java:748)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "Service Thread" #7 daemon prio=9 os_prio=0 tid=0x00007f05980b7000 nid=0x36 runnable [0x0000000000000000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: RUNNABLE
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "C1 CompilerThread1" #6 daemon prio=9 os_prio=0 tid=0x00007f05980b3800 nid=0x35 waiting on condition [0x0000000000000000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "C2 CompilerThread0" #5 daemon prio=9 os_prio=0 tid=0x00007f05980b1800 nid=0x34 waiting on condition [0x0000000000000000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "Signal Dispatcher" #4 daemon prio=9 os_prio=0 tid=0x00007f05980b0000 nid=0x33 waiting on condition [0x0000000000000000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "Finalizer" #3 daemon prio=8 os_prio=0 tid=0x00007f059807d800 nid=0x32 in Object.wait() [0x00007f059cefd000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] java.lang.Thread.State: WAITING (on object monitor)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076ab78e78> (a java.lang.ref.ReferenceQueue$Lock)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:165)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:216)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "Reference Handler" #2 daemon prio=10 os_prio=0 tid=0x00007f059807b000 nid=0x31 in Object.wait() [0x00007f059cffe000]
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.Object.wait(Object.java:502)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] - locked <0x000000076ab79030> (a java.lang.ref.Reference$Lock)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "VM Thread" os_prio=0 tid=0x00007f0598071000 nid=0x30 runnable
2018-11-23T15:41:15.419+01:00 [APP/PROC/WEB/0] [OUT] "VM Periodic Task Thread" os_prio=0 tid=0x00007f05980b9800 nid=0x37 waiting on condition
2018-11-23T15:41:15.472+01:00 [APP/PROC/WEB/0] [OUT] JNI global references: 1376931
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] Heap
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] def new generation total 54400K, used 1006K [0x0000000722c00000, 0x0000000726700000, 0x000000076ab50000)
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] eden space 48384K, 2% used [0x0000000722c00000, 0x0000000722cfbb90, 0x0000000725b40000)
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] from space 6016K, 0% used [0x0000000725b40000, 0x0000000725b40000, 0x0000000726120000)
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] to space 6016K, 0% used [0x0000000726120000, 0x0000000726120000, 0x0000000726700000)
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] tenured generation total 120672K, used 64693K [0x000000076ab50000, 0x0000000772128000, 0x00000007faa00000)
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] the space 120672K, 53% used [0x000000076ab50000, 0x000000076ea7d510, 0x000000076ea7d600, 0x0000000772128000)
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] Metaspace used 92462K, capacity 93352K, committed 94292K, reserved 1132544K
2018-11-23T15:41:15.473+01:00 [APP/PROC/WEB/0] [OUT] class space used 10834K, capacity 11043K, committed 11136K, reserved 1048576K
2018-11-23T15:41:20.416+01:00 [APP/PROC/WEB/0] [ERR] jvmkill killing current process
```

**Comment by woonsanko** [comment]
Hi @fdeberle ,

Two things in my mind:
- could you try with `-t 180` in your push command if you didn't? (ref: https://github.com/woonsanko/hippo-on-spring-boot#deploying-to-the-cloud-eg-cloud-foundry-heroku-etc) Your first logs shows less than 1 min diff between the first line and the last. So, perhaps PCF used 1 min timeout, the default value, if not specified with `-t 180` even if you defined in the manifest.
- second, how does it work when running it locally? (ref: https://github.com/woonsanko/hippo-on-spring-boot#run) Is it working fine locally with `test.sh`?

Woonsan

**Comment by fdeberle** [comment]
Hi @woonsanko,

I tried `-t 180` on the command line—no effect, same error log.
Running locally works as expected.

I am running out of ideas. Using the DeferredInitDelegatingServlet sounded reasonable, but it did not help. The first request on `/cms` crashes the app.

Strange detail: if I reduce the `deployableWars` to `cms.war`, it runs on PCF. But we need the [REST API](https://www.onehippo.org/library/concepts/rest/content-rest-api/document-collection-resource.html) and therefor also `site.war`.

**Comment by woonsanko** [comment]
Hi @fdeberle ,
Hmm... the stack trace doesn't look weird - just looks like PCF wanted to kill it for some reason (e.g, timeout). The log messages doesn't seem helpful enough at the moment. :(
My thinking is, as long as it work locally by executing the 'executable' jar, it should work on PCF too. Also, with `JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '[enabled: false]'`, PCF wouldn't do anything specific for JNDI resources for example.
Not sure if it's related to 12.2 vs 12.5...  Was it successful to deploy this demo project (v12.2 based atm) on PCF, could you try to upgrade this demo to 12.5 locally and deploy that on PCF to see if it's related to 12.5 upgrade issue? The pom.xml will probably need to be changed only 2-3 versions though.
I'm sorry, but I'm going on holidays and will come back after a while. So, I cannot test it myself unfortunately at the moment..

Kind regards,

Woonsan
